fairlearn/postprocessing/_threshold_operation.py

```diff
@@ -18,7 +18,7 @@ class ThresholdOperation:
     """
 
     def __init__(self, operator, threshold):
-        if operator not in [">", "<"]:
+        if operator not in [">", "<"]:          # NOTE for PR: sklearn uses >= for ROC threshold; see: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html
```

Comment:
I think the thresholds in sklearn are the predicted scores for one or more instances ([see code here](https://github.com/scikit-learn/scikit-learn/blob/364c77e04/sklearn/metrics/_ranking.py#L794C22-L794C29)), so using `>` instead of `>=` would mean flipping the predictions for some portion of samples.

If it's possible to change it here that would be great!

If not, is there any fairlearn code to build the (fpr, tpr, threshold) ROC triplets? I could switch to using that.

Reply:
Take a look at `_tradeoff_curve` here:
https://github.com/fairlearn/fairlearn/blob/main/fairlearn/postprocessing/_tradeoff_curve_utilities.py